doc/api/vm.md

```diff
@@ -1573,6 +1573,90 @@ inside a `vm.Context`, functions passed to them will be added to global queues,
 which are shared by all contexts. Therefore, callbacks passed to those functions
 are not controllable through the timeout either.
 
+## Local Worker
```

Comment:
Can we drop this, move the Stability note under the `NodeRealm` class and increase the subsequent header sizes.